packages/core/src/lib/__tests__/radio.spec.ts

```diff
@@ -1,142 +1,108 @@
 import { describe, it, expect, vi } from 'vitest';
-import { render, screen, waitFor } from '@testing-library/svelte';
+import { render, screen } from '@testing-library/svelte';
+import Radio from './radio.spec.svelte';
 import userEvent from '@testing-library/user-event';
-import { Radio } from '$lib';
-import * as MDI from '@mdi/js';
 
 describe('Radio', () => {
+  const common = {
+    name: 'Test radios',
+    options: ['Option 1', 'Option 2', 'Option 3'],
+    selected: 'Option 1',
+  };
+
   it('Renders radio options correctly', () => {
-    render(Radio, { options: ['Option1', 'Option2'], selected: 'Option1' });
-    expect(screen.getByText('Option1')).toBeVisible();
-    expect(screen.getByText('Option2')).toBeVisible();
+    render(Radio, common);
+    expect(screen.getByLabelText('Option 1')).toBeVisible();
+    expect(screen.getByLabelText('Option 2')).toBeVisible();
+    expect(screen.getByLabelText('Option 3')).toBeVisible();
   });
 
   it('Marks the selected option correctly', () => {
```

Comment:
Should this test be checking more HTML semantics, like which element is `checked`?

Reply:
Sure, I just tried to take the previous tests and convert them over to make sure things still "worked" as expected, but I agree we should focus more on the html then the classes. I will take a pass at these tests.